.circleci/config.yml

```diff
@@ -29,6 +29,29 @@ jobs:
       - store_test_results:
           path: test-results
 
+  test-local-environment:
+    executor: docker/machine
+    steps:
+      - checkout
+      - attach_workspace:
+          at: artifacts
+      - docker/install-docker-compose
+      - run:
+          name: Load Docker image artifact from previous job
+          command: docker load -i artifacts/telemetry-airflow.tar
+      - run:
+          name: Override docker-compose.yaml with artifact
+          command: |
+            sed -i "s/  build: ./  image: $CIRCLE_PROJECT_REPONAME:${CIRCLE_SHA1:0:9}/g" docker-compose.yml
```

Comment:
I don't believe this line will be changed under normal circumstances. The expected behaviour for local development is to build the local image. I was looking into using docker-compose overrides, but it won't work because the key is different (`image` vs `build`). If someone does change the line, we'll have a CI failure and it should be trivial to figure out the issue.